inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupServiceImpl.java

```diff
@@ -465,7 +465,7 @@ public Boolean delete(String groupId, String operator) {
         // logically delete the associated extension info
         groupExtMapper.logicDeleteAllByGroupId(groupId);
 
-        if (GroupStatus.allowedLogicDelete(GroupStatus.forCode(entity.getStatus()))) {
+        if (GroupStatus.notAllowedDelete(GroupStatus.forCode(entity.getStatus()))) {
             streamService.logicDeleteAll(groupId, operator);
```

Comment:
Maybe, `GroupStatus.notAllowedDelete(GroupStatus.forCode(entity.getStatus()))`  modify to `!GroupStatus.notAllowedDelete(GroupStatus.forCode(entity.getStatus()))`

Reply:
Fixed.